composer.json

```diff
@@ -15,9 +15,9 @@
     "require": {
         "php": "^7.0",
         "matthiasnoback/symfony-config-test": "^3.0",
-        "symfony/dependency-injection": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0",
-        "symfony/yaml": "^2.7|^3.0"
+        "symfony/dependency-injection": "^2.7|^3.0|^4.0",
+        "symfony/config": "^2.7|^3.0|^4.0",
```

Comment:
If you require only supported 2.x versions, I would also bump the `3.x` branch to `^3.3`.